src/ConnectedMode/Migration/Interfaces.cs

```diff
@@ -95,21 +95,44 @@ internal interface IFileCleaner
     /// <remarks>Required by <see cref="IFileCleaner"/> to identify the references to remove</remarks>
     internal class LegacySettings
     {
-        public LegacySettings(string partialRuleSetPath, string partialSonarLintXmlPath)
+        public LegacySettings(
```

Comment:
I needed to add a path to the `.sonarlint` folder to settings so it can be deleted.

@georgii-borovinskikh-sonarsource pointed out that VB and CSharp have separate ruleset and SonarLint.xml paths, so I added those at the same time.